packages/common-css/README.md

```diff
@@ -1,3 +1,37 @@
-# Fundamental Library Common CSS Package
+Fundamental Library Common CSS Package
 
-This package contains Common CSS files. More documentation to come soon...
+## What is Fundamental Library Common CSS?
+
+Fundamental Library Common CSS Package holds all the CSS style rules of the library that are recognized by older browsers—as well as the newer browsers. It creates the style rules for the basic elements on a page, such as a (link) elements, p (paragraph) elements, h1 (header) elements etc. The package is based on the DRY (Don't Repeat Yourself) methodology which allows the developers to not repeat the same CSS rules overall in the library.
```

Comment:
I would say that the common-css is the "set of utility CSS classes similar to Tailwind CSS to create own components in SAP look but without manually writing CSS".

Reply:
I don't think we need to explain out technologies through other technologies.

Reply:
Ignore this one... the updated common css read me is in the other pr